maestro-domain/src/main/java/bio/overture/maestro/domain/api/DefaultIndexer.java

```diff
@@ -559,12 +695,22 @@ private FileConflict toFileConflict(FileCentricDocument f1, FileCentricDocument
     }
 
     private Mono<IndexResult>
-        batchUpsertFilesAndCollectFailures(Tuple2<FailureData, List<FileCentricDocument>> failureDataAndFileListTuple) {
-        return this.batchUpsert((failureDataAndFileListTuple._2()))
-            .map(upsertResult -> reduceIndexResult(IndexResult.builder()
-                .failureData(failureDataAndFileListTuple._1())
-                .build(), upsertResult)
-            );
+        batchUpsertFilesAndCollectFailures(Tuple2<FailureData, List<FileCentricDocument>> tuple) {
+        return this.batchUpsert( tuple._2() )
+                .map( upsertResult -> reduceIndexResult(
```

Comment:
the indentation tab for chain calls is set to 8 in your intellj can you set it to 4 please
![image](https://user-images.githubusercontent.com/8655252/79770341-47217a80-82fb-11ea-8da9-6005153b203b.png)